src/Native/jitinterface/jitinterface.cpp

```diff
@@ -67,37 +67,37 @@ class EEMemoryManager
     {
     }
 
-    virtual void __stdcall QueryInterface() { NotImplemented(); }
-    virtual void __stdcall AddRef() { NotImplemented(); }
-    virtual void __stdcall Release() { NotImplemented(); }
+    virtual void DEFAULT_CALL_CONV QueryInterface() { NotImplemented(); }
```

Comment:
I would mark all these `STDMETHODCALLTYPE` and then `#define STDMETHODCALLTYPE` somewhere, so that the callconv marked is same.

Reply:
I finally understood. STDMETHODCALLTYPE as it's in CoreCLR. Then yes, it's more correct.